resultbrowser/src/js/resultbrowser.js

```diff
@@ -95,211 +183,233 @@
-                var s = $requestText.text();
-                // CSS
-                if($requestText.hasClass('css')) {
-                    try {
-                        s = css_beautify(s);
-                    }
-                    catch(e){}
+        // init listeners for beautify button
+        beautify.addEventListener("click", function () {
+            let s = getText(requestText);
+            // CSS
+            if (requestText.classList.contains("css")) {
+                try {
+                    s = css_beautify(s);
                 }
-                // Javascript / JSON
-                else if($requestText.hasClass('javascript')) {
-                    try {
-                        s = js_beautify(s);
-                    }
-                    catch(e){}
+                catch (e) { }
+            }
+            // Javascript / JSON
+            else if (requestText.classList.contains("javascript")) {
+                try {
+                    s = js_beautify(s);
                 }
-                // HTML
-                else if($requestText.hasClass('html') || $requestText.hasClass('xml')) {
-                    try {
-                        s = html_beautify(s, {
-                            preserve_newlines : false,
-                            wrap_line_length : 0
-                        });
-                    }
-                    catch(e){}
+                catch (e) { }
+            }
+            // HTML
+            else if (requestText.classList.contains("html") || requestText.classList.contains('xml')) {
+                try {
+                    s = html_beautify(s, {
+                        preserve_newlines: false,
+                        wrap_line_length: 0
+                    });
                 }
-                $requestText.text(s);
-            }).removeAttr('disabled');
-        }
-
-        $('#selectResponseContent').unbind("click").click(function() {
-          document.getSelection().selectAllChildren($requestText.get(0));
+                catch (e) { }
+            }
+            setText(requestText, s);
         });
 
+        const selectResponseContent = getElementById("selectResponseContent");
+
+        const listener = function () {
+            document.getSelection().selectAllChildren(requestText);
+        }
+
+        selectResponseContent.removeEventListener("click", listener);
+        selectResponseContent.addEventListener("click", listener);
 
         // menu button
-        $menuIcon.click(showMenu);
-        $(document).click(function(e){
-            var x = e.target;
-            if($(x).parents('#menu').length === 0 && x.id != "menu-icon") {
-                if($menu.hasClass("open")) {
+        menuIcon.addEventListener("click", showMenu);
+        document.addEventListener("click", function (e) {
+            const x = e.target;
+            if (getParents(x).filter(parent => parent.id == "menu").length === 0 && x.id != "menu-icon") {
```

Comment:
For consistency reasons I'd like to stay with traditional function declarations (no arrow functions).